documentation/devon4j.asciidoc

```diff
@@ -257,6 +262,10 @@ include::guide-jms.asciidoc[leveloffset=+3]
 
 include::guide-text-search.asciidoc[leveloffset=+3]
 
+<<<<
+
+include::guide-lombok.asciidoc[leveloffset=+3]
```

Comment:
Linking this guide here would display it below _Spring_ on the devonfw website. But I think it should be displayed in the _General_ section right?